.github/workflows/push-content.yml

```
@@ -0,0 +1,21 @@
+name: Push content
+
+on:
+  workflow_dispatch:
+  push:
+    branches: 
+      - master
+    paths:
+      - 'modules/ROOT/**'
+      - 'antora.yml'
+jobs:
+  triggerJob:    
+    runs-on: ubuntu-latest    
```

Let's use a specific version to avoid surprise when github decide to change the latest version
```suggestion
    runs-on: ubuntu-20.04
```